CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ Mapbox welcomes participation and contributions from everyone.
 ## Mapbox Navigation SDK 2.0.0-beta.3 - March 26, 2021
 Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
 
+### Important note
```

Comment:
Could also be a `### Know issues` at the bottom as we used to do.

Reply:
I'll make align with previous